firmware_configurations/klipper/RealDeuce/MKS-Makerbase/Monster8_v1.0_003/Voron2_Monster8_Config.cfg

```diff
@@ -0,0 +1,531 @@
+stealthchop_threshold: 0
+
+#! In Driver3:E0 Position (Z1 Stepper - Rear Left)
+[stepper_z1]
+step_pin: PB5
+dir_pin: PB4
+enable_pin: !PB6
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+[tmc2209 stepper_z1]
+uart_pin: PB3
+interpolate: True
+run_current: 0.8                # Max 1.4 (tmc2209 limit)
+hold_current: 0.5
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+#! In Driver4:E1 Position (Z2 Stepper - Rear Right)
+[stepper_z2]
+step_pin: PD6
+dir_pin: PD5
+enable_pin: !PD7
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+[tmc2209 stepper_z2]
+uart_pin: PD0
+interpolate: true
+run_current: 0.8                # Max 1.4 (tmc2209 limit)
+hold_current: 0.5
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+## In Driver5:E2 Position (Z3 Stepper - Front Right)
+[stepper_z3]
+step_pin: PD2
+dir_pin: PD1
+enable_pin: !PD3
+rotation_distance: 40
+gear_ratio: 80:16
+microsteps: 16
+
+[tmc2209 stepper_z3]
+uart_pin: PD4
+interpolate: true
+run_current: 0.8                # Max 1.4 (tmc2209 limit)
+hold_current: 0.5
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+#####################################################################
+#   Extruder
+#####################################################################
+
+#! In Driver7:E4 Position (Extruder)
+[extruder]
+step_pin: PD13
+dir_pin: PD12
+enable_pin: !PD14
+
+## Update value below when you perform extruder calibration
+## If you ask for 100mm of filament, but in reality it is 98mm:
+## rotation_distance = <previous_rotation_distance> * <actual_extrude_distance> / 100
+##  22.6789511 is a good starting point for Bondtech 5mm gears
+##  34.37086 for Bondtech 8mm gears (Galileo)
+rotation_distance: 22.6789511   #Bondtech 5mm Drive Gears
+## Update Gear Ratio depending on your Extruder Type
+## Use 50:17 for Afterburner/Clockwork (BMG Gear Ratio)
+## Use 80:20 for M4, M3.1
+## Use 7.5:1 for Galileo
+gear_ratio: 50:17               #BMG Gear Ratio
+microsteps: 16
+full_steps_per_rotation: 200    #200 for 1.8 degree, 400 for 0.9 degree
+nozzle_diameter: 0.400
+filament_diameter: 1.75
+#! In HE0 Position (Hotend heater)
+heater_pin: PB1
+## Validate the following thermistor type to make sure it is correct
+sensor_type: ATC Semitec 104GT-2
+#! In TH0 Position (Hotend thermistor)
+sensor_pin: PC1
+min_temp: 10
+max_temp: 270
+max_power: 1.0
+min_extrude_temp: 170
+control = pid
+pid_kp = 26.213
+pid_ki = 1.304
+pid_kd = 131.721
+## Try to keep pressure_advance below 1.0
+#pressure_advance: 0.05
+## Default is 0.040, leave stock
+#pressure_advance_smooth_time: 0.040
+
+[tmc2209 extruder]
+uart_pin: PD11
+interpolate: false
+run_current: 0.5
+hold_current: 0.4
+sense_resistor: 0.110
+stealthchop_threshold: 0
+
+#####################################################################
+#   Bed Heater
+#####################################################################
+[heater_bed]
+#! In H-BED position (SSR)
+heater_pin: PB10
+sensor_type: NTC 100K MGB18-104F39050L32
```

Comment:
also comment out this line to force user to pick. Generally the bed should be `Generic 3950` instead of this type.